.github/workflows/build-wheel-linux-arm64.yaml

```diff
@@ -14,12 +14,29 @@ on:
     # Thursdays we test the standalone plugin
     - cron: '35 4 * * 4'
   workflow_dispatch:
+    inputs:
+      build_standalone_plugin:
+        description: 'Build the standalone plugin wheel'
+        required: false
+        default: 'false'
   workflow_call:
+    inputs:
+      build_standalone_plugin:
```

Comment:
We don't seem to set this input in another action, is it just there for the future?

Reply:
Sorry I was just wondering if you intended this value to be set by another workflow (on call), but now I understand you primarily want the value to be set at all and contrast it when it is unset.